ruleset/rules/0755-office365_rules.xml

```diff
@@ -0,0 +1,928 @@
+        <field name="office_365.RecordType" type="osregex">^68$</field>
+        <description>Office 365: Events tracked by the Communication compliance offensive language model.</description>
+        <options>no_full_log</options>
+        <group>ComplianceSupervisionExchange</group>
+    </rule>
+
+    <rule id="91597" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^69$</field>
+        <description>Office 365: Events related to the customer key encryption service.</description>
+        <options>no_full_log</options>
+        <group>CustomerKeyServiceEncryption</group>
+    </rule>
+
+    <rule id="91598" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^70$</field>
+        <description>Office 365: Events related to sensitivity labels applied to Office documents.</description>
+        <options>no_full_log</options>
+        <group>OfficeNative</group>
+    </rule>
+
+    <rule id="91599" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^71$</field>
+        <description>Office 365: Auto-labeling events in SharePoint.</description>
+        <options>no_full_log</options>
+        <group>MipAutoLabelSharePointItem</group>
+    </rule>
+
+    <rule id="91600" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^72$</field>
+        <description>Office 365: Auto-labeling policy events in SharePoint.</description>
+        <options>no_full_log</options>
+        <group>MipAutoLabelSharePointPolicyLocation</group>
+    </rule>
+
+    <rule id="91601" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^73$</field>
+        <description>Office 365: Teams Shifts events.</description>
+        <options>no_full_log</options>
+        <group>MicrosoftTeamsShifts</group>
+    </rule>
+
+    <rule id="91602" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^75$</field>
+        <description>Office 365: Auto-labeling events in Exchange.</description>
+        <options>no_full_log</options>
+        <group>MipAutoLabelExchangeItem</group>
+    </rule>
+
+    <rule id="91603" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^76$</field>
+        <description>Office 365: Briefing email events.</description>
+        <options>no_full_log</options>
+        <group>CortanaBriefing</group>
+    </rule>
+
+    <rule id="91604" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^77$</field>
+        <description>Office 365: Events related to performing search queries in SharePoint and Exchange.</description>
+        <options>no_full_log</options>
+        <group>Search</group>
+    </rule>
+
+    <rule id="91605" level="5">
```

Comment:
I think this level should be higher since these alerts come from Windows Defender.

Reply:
Done